src/main/resources/application.properties

```diff
@@ -14,7 +14,8 @@ email=
 password=
 path=
 projectId=
+syncXray=
```

Comment:
I think we should push the default value to this prop

Reply:
updated